test/unittest/main.cpp

```diff
@@ -25,11 +25,46 @@
 
 #include <gtest/gtest.h>
 
+#include "blas_test.hpp"
 #include "blas_test_macros.hpp"
+#include "clara.hpp"
 
-int main(int argc, char *argv[]) {
+struct Args args{};
+
+namespace {
+/**
+ * @fn parse_args
+ * @brief Returns a structure containing the data extracted from the
+ * command-line arguments.
+ */
+void parse_args(int argc, char** argv) {
```

Comment:
If a user does not pass a device type, the default selector should be picked

Reply:
Should be done now, both the SYCL benchmarks and the unit tests now will use the default selector if args.device is empty.